docs/proposals/koordlet/20221018-koordlet-support-cgroups-v2.md

```diff
@@ -522,7 +522,7 @@ func (p *Plugin) applyContainerCFSQuota(podMeta *statesinformer.PodMeta, contain
 		}
 
 		// for container-level
-		containerDir, err := util.GetContainerCgroupPathWithKube(podMeta.CgroupDir, containerStat)
+		containerDir, err := util.GetContainerCgroupParentDir(podMeta.CgroupDir, containerStat)
```

Comment:
how about save the dir in podMeta, so that we don't need to change lots of interfaces

Reply:
done.